### DISCUSSION TITLE
PLS how remove this line

### DISCUSSION THREAD

**Discussion by Mopslik** [black7375/Firefox-UI-Fix#887] [General]
Hello, i like your old style Firefox, new is bad for me. Please, how remove this line in active tab.
![Firefox](https://github.com/black7375/Firefox-UI-Fix/assets/169046442/2dde35ab-153c-43f0-9aac-7f8c57facbb3)

I modify this in user.js:
user_pref("userChrome.icon.panel_full",                         true); // Original, Proton
user_pref("userChrome.fullscreen.overlap",                    false);
user_pref("userChrome.fullscreen.show_bookmarkbar", false);
user_pref("userChrome.tab.static_separator",                 false); // Photon
user_pref("userChrome.decoration.animate",                 false);

I found this but dont work for me:
userChrome.tab.photon_like_contextline                       false);

Thanks for help, bye.

**Comment by black7375** [comment]
Are you using `userChrome.tab.connect_to_window` to `true`?
If not, set its to `true`.

If the problem occurs even if it is `true`, try updating the theme to the latest version.
If you're still having trouble, it may be a bug in specific OS or theme.
Please tell me the OS, theme with URL, and UI density you are using.

**Reply by Mopslik** [reply]
userChrome.tab.connect_to_window - was true.

If i change in user.js to false:
user_pref("userChrome.tab.static_separator",          false); // Photon 
bad line is visible.

I dont like fade animation. 
I used theme 8.6.1. 
https://github.com/black7375/Firefox-UI-Fix/archive/refs/heads/photon-style.zip
Windows 10 1909. Compact UI density.

**Reply by Mopslik** [reply]
Firefox 125.0.3 64-bit

**Reply by Mopslik** [reply]
Sorry , i post bad code.
If i change in user.js to false:
user_pref("userChrome.decoration.animate",        false);
bad line is visible.

I dont like fade animation.

**Reply by black7375** [reply]
There seems to have been a communication mistake in the theme URL.
For example, I needed information for a theme I install from the add-on store, like this(This is because problems may occur depending on the background image or transparency):
https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/


When I have some time to rest, I will try to reproduce it according to your information.

**Reply by Mopslik** [reply]
Good job, the problem is in theme, i used this:
https://addons.mozilla.org/sk/firefox/addon/proton-to-photon/

When i change to other, bad line is gone. But i like photon blue.
In this theme is line much thinner.
https://addons.mozilla.org/sk/firefox/addon/photon-colors/

Pictures:
When is true and my old theme: user_pref("userChrome.decoration.animate",        true);
![1](https://github.com/black7375/Firefox-UI-Fix/assets/169046442/2716d373-9a2d-4c65-9421-428ec65c881b)

When is false and my old theme: user_pref("userChrome.decoration.animate",        false);
![2](https://github.com/black7375/Firefox-UI-Fix/assets/169046442/00ebae32-8d34-44c9-9106-99aa9984b0cf)

When is true and my NEW theme: user_pref("userChrome.decoration.animate",        true);
![3](https://github.com/black7375/Firefox-UI-Fix/assets/169046442/bd3258a1-c103-4b3e-8f9f-27a33511bbf7)
But this theme has other problems for me. :)
I dont lie new Firefox design... :/

**Reply by black7375** [reply]
The latest commit fixes this issue!!!

**Reply by Mopslik** [reply]
Hello, great news, thanks. But i temporarily rolled back to Firefox 114, i found other problems that are not related to your theme. Once again, thanks for your work. Bye

**Reply by Mopslik** [reply]
I tried your changes and line is gone. Thanks for good job. Theme 8.6.1 and Firefox 126. Bye.
